sp800-63c/sec5_assertions.md

```diff
@@ -41,7 +43,9 @@ Note that mere possession of a bearer assertion or reference is not always enoug
 
 #### 5.1.2. Holder-of-Key Assertions
 
-A holder-of-key assertion contains a reference to a symmetric key or a public key (corresponding to a private key) possessed by and representing the subscriber.  To establish a holder-of-key binding of an assertion to a claimant, the RP SHALL require the claimant to prove possession of the key that is referenced in the assertion in addition to presentation of the assertion itself. An assertion containing a reference to a key held by the subscriber for which key possession has not been proven SHALL be considered a bearer assertion.
+A holder-of-key assertion contains a reference to a symmetric key, or a public key that corresponds to a private key, possessed by and representing the subscriber.  To establish a holder-of-key binding of an assertion to a claimant, the RP SHALL require the claimant to prove possession of the key that is referenced in the assertion in addition to presentation of the assertion itself. An assertion containing a reference to a key held by the subscriber for which key possession has not been proven SHALL be considered a bearer assertion.
+
+>**MG: How does this last sentence work? Where is the holder-of-key in that scenario?**
```

Comment:
> Mike: maybe it’s just the sentence structure because, on a quick and not overly critical read, the first sentence says you just need the reference, not also proof of possession.